flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/TestBase.java

```diff
@@ -122,7 +123,23 @@ protected void assertSameElements(String message, Iterable<Row> expected, Iterab
    * @param ifExists If we should use the 'IF EXISTS' when dropping the catalog
    */
   protected void dropCatalog(String catalogName, boolean ifExists) {
-    sql("USE CATALOG default_catalog");
+    sql("USE CATALOG %s", DEFAULT_CATALOG_NAME);
     sql("DROP CATALOG %s %s", ifExists ? "IF EXISTS" : "", catalogName);
   }
+
```

Comment:
this is a change introduced by 1.19. With the refactoring, this is needed for 1.18 now.